tests/components/image_upload/test_init.py

```diff
@@ -49,7 +49,12 @@ async def test_upload_image(
 
         tempdir = pathlib.Path(tempdir)
         item_folder: pathlib.Path = tempdir / item["id"]
-        assert (item_folder / "original").read_bytes() == TEST_IMAGE.read_bytes()
```

Comment:
We should keep this line as it checks that the file is saved correctly.

You could create a variable for `TEST_IMAGE.read_bytes()` so you read it only once

Reply:
I found it redundant since when we return the original image, we check its contents already. Are you sure we need to check it twice, once on the file system and then via the API?

Reply:
Yes I would keep and check it twice

Reply:
I added it 👍